### PR TITLE
VFM-7455 Bump logging levels for sparse events

### PR DESCRIPTION
**Pull request by reed-atakama**
Bump one-off log lines. Don't bump regularly occurring log lines higher than debug.

Three areas that should be logged: 1) Cloudsync errors raised 2) conflict resolution 3) punts

### CODE REVIEW

File: cloudsync/sync/manager.py [hunk 1]
```diff
@@ -1304,22 +1304,22 @@ def conflict_rename(self, side, path):
...
         if sync[changed].path or (sync[changed].exists == EXISTS):
             translated_path = self.translate(synced, sync[changed].path)
             if not translated_path:
                 if sync[changed].sync_path:  # This entry was relevent, but now it is irrelevant
-                    log.debug(">>>Removing remnants of file moved out of cloud root")
+                    log.info(">>>Removing remnants of file moved out of cloud root")
                     sync[changed].exists = TRASHED  # This will discard the ent later
                 else:  # we don't have a new or old translated path... just irrelevant so discard
-                    log.log(TRACE, ">>>Not a cloud path %s, ignoring", sync[changed].path)
+                    log.info(">>>Not a cloud path %s, ignoring", sync[changed].path)
```
Comment (colatkinson) [inline cloudsync/sync/manager.py:1322]: This line looks like it will get triggered every time a file outside of the bed gets changed. Is that accurate?

Comment (reed-atakama) [reply cloudsync/sync/manager.py:1322]: Ya I think you might be right

File: cloudsync/sync/manager.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ def download(self) -> str:
                     self.provider.download(self.info.oid, f)
                 os.rename(self.__temp_file + ".tmp", self.__temp_file)
             except Exception as e:
-                log.debug("error downloading %s", e)
+                log.error("error downloading %s", e)
```
Comment (earonesty) [inline cloudsync/sync/manager.py:82]: info or warning logging is sufficient for any issues that are related to "normal operation", but could be a provider, network, filesystem problem. 

probably we could have OSError be a warning, and Exception be an error... but I'm fine with warning on most things.

in the near future, errors will be actionable items

File: cloudsync/sync/manager.py [hunk 3]
```diff
@@ -451,14 +451,14 @@ def download_changed(self, changed, sync):
...
-            log.debug("file not found %s", sync[changed].path)
+            log.warning("file not found %s", sync[changed].path)
             sync[changed].clean_temp()
             return False
         except PermissionError as e:
             raise ex.CloudTemporaryError("download or rename exception %s" % e)
 
         except ex.CloudFileNotFoundError:
-            log.debug("download from %s failed fnf, switch to not exists",
+            log.error("download from %s failed fnf, switch to not exists",
```
Comment (earonesty) [inline cloudsync/sync/manager.py:461]: warning

File: cloudsync/sync/manager.py [hunk 4]
```diff
@@ -601,13 +601,13 @@ def upload_synced(self, changed, sync):
                 sync, synced, exists=True, oid=info.oid, path=sync[synced].sync_path)
             return True
         except FileNotFoundError:
-            log.info("FNF during upload %s:%s", sync[synced].sync_path, sync[changed].temp_file)
+            log.error("FNF during upload %s:%s", sync[synced].sync_path, sync[changed].temp_file)
```
Comment (earonesty) [inline cloudsync/sync/manager.py:604]:
```suggestion
            log.warning("FNF during upload %s:%s", sync[synced].sync_path, sync[changed].temp_file)
```

File: cloudsync/sync/manager.py [hunk 5]
```diff
@@ -558,10 +558,10 @@ def mkdir_synced(self, changed, sync, translated_path):
 
         if ents:
             if sync.priority <= 0:
-                log.debug("punt mkdir")
+                log.info("punt mkdir %s", translated_path)  # pragma: no cover
```
Comment (reed-atakama) [inline cloudsync/sync/manager.py:561]: I couldn't figure out how to replicate this scenario

**Pull request merged**